### ISSUE TITLE
[Feature Request] Alerts - Options for not coloring text

### ISSUE THREAD

**Issue by megaroeny** [vuetifyjs/vuetify#8754]
### Problem to solve
I would like to see the option of having the combination of an Outlined alerts + the Border, but not make the text colored. This can pose a color contrast/accessibility issue if the color palette fails contrast. Yellow is a perfect example - that will _always_ fail color contrast on a white background or white text on a yellow background. 

### Proposed solution
Provide another option to not color the text in an Alert. See what our Alerts look like in our design system in Figma below:

![image](https://user-images.githubusercontent.com/1559447/63792030-b6f57780-c8ca-11e9-921c-5b1ca0771e0e.png)

OR what if you have an option to add an outline to a Normal alert, in addition to the `colored-border` option?

**Comment by megaroeny** [comment]
Any updates or possibility of revisiting this?

**Comment by MajesticPotatoe** [comment]
Currently most of our efforts are geared toward v3 as there is a lot that needs to go into it. However we highly encourage the community to submit PR's. I believe due to the delay in vue 3 there is a possibility of a v2.3 which would welcome this feature.

There's probably a few ways you can handle this.
One way to do it like your image:
https://codepen.io/majesticpotatoe-the-bashful/pen/yLNoEpw

You can also easily handle this with sass vars

**Comment by megaroeny** [comment]
Got it! Sounds good and excited about V3. 

A little off topic but: We just had a meeting earlier this week where some of us had mentioned possibly using storybook to document our design system. So we're excited about that opportunity. 

Thanks for the quick response!

**Comment by johnleider** [comment]
![image](https://user-images.githubusercontent.com/9064066/124176081-0c28e400-da74-11eb-93e4-0551dec4e901.png)

The recommended way to do this in v3 is:

```vue
<template>
  <v-app>
    <div class="ma-12 pa-12">
      <v-alert
        border="start"
        class="border-opacity-100"
        closable
        density="comfortable"
        icon="mdi-information-outline"
        prominent
        type="warning"
        variant="outlined"
      >
        <div class="text-black">
          Lorem ipsum dolor sit amet consectetur adipisicing elit.
        </div>

        <template #close>
          <v-icon color="black">mdi-close</v-icon>
        </template>
      </v-alert>
    </div>
  </v-app>
</template>
```

fixed in #13855

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
